web_searchbar_full_width/README.rst

```diff
@@ -0,0 +1,103 @@
+====================
+Full width searchbar
```

Comment:
@bizzappdev Two Titles is Different?

Second: https://github.com/OCA/web/pull/1218/files#diff-b3dd890285fb4d4e41c8ae29c21f33a2R29